dev.py

```diff
@@ -426,17 +427,63 @@ def install_project(show_build_log):
             raise
     elapsed = datetime.datetime.now() - start_time
 
-    if ret == 0:
-        print("Installation OK")
-    else:
+    if ret != 0:
         if not show_build_log:
             with open(log_filename, 'r') as f:
                 print(f.read())
         print("Installation failed! ({0} elapsed)".format(elapsed))
         sys.exit(1)
+
+    # On Windows we also need openblas lib installed
+    if platform.system() == 'Windows':
+        ret = copy_openblas()
+    if ret != 0:
+        print("OpenBLAS copy failed!")
+        sys.exit(1)
+    print("Installation OK")
     return
 
 
+def import_module(mod_name, mod_path):
+    # https://bugs.python.org/msg255901
```

Comment:
That's 5 years old and not particularly relevant info - can you remove it or just spell out what the reader needs to know here?